cmd/cue/cmd/testdata/script/issue2969.txtar

```diff
@@ -0,0 +1,37 @@
+# Test for go get issue with time.Duration returning invalid numeric.
+#
+# cue get go
```

Comment:
remove this line? it doesn't seem to add anything

Reply:
done in https://github.com/cue-lang/cue/commit/fa8b5594062955ab4b7a28676176cd55dc273e92